qiskit_nature/operators/second_quantization/spin_op.py

```diff
@@ -406,7 +406,8 @@ def reduce(self, atol: Optional[float] = None, rtol: Optional[float] = None) ->
         new_coeff = coeff_list[non_zero]
         return SpinOp((new_array, new_coeff), spin=self.spin)
 
-    def to_list(self) -> List[Tuple[str, complex]]:
+    # pylint: disable=arguments-differ
```

Comment:
As per my comment above, this should not be necessary. We can simply not use the argument.

Reply:
So you recommend meaningless arguments. That is one direction. (# pylint: disable=unused-argument)

Reply:
Well since you add the argument on the interface anyways, I would rather make the code in the `QubitMapper` not case-specific.
Adding this exception here causes complexity in the `QubitMapper` *and* a lint exception here. Having an argument which is ignored here and in the vibrational case only causes a lint exception (which may even get remove in the future)

Reply:
I reverted the interface. Since FermionicOp is an exception, I shouldn't have specified it in the super class. In the future, I think it would be better to be able to map sparse label directly.

Reply:
Okay I see. I absolutely agree that we need to be able to map sparse labels directly in the future. Until then, let's use this workaround with the `isinstance` check inside of the `QubitMapper`. @woodsp-ibm is this okay with you, too?